type.py

```diff
@@ -86,21 +86,19 @@ def get_configs(self):
         return vars(self)
 
 
-""" Run Configs """
+""" Experiment """
 
 
 @dataclass
-class RunConfig:
+class Experiment:
+    project_name: str
     run_name: str  # Get's appended as a prefix before the pipeline name
     train: bool  # Weather the run should do training
     dataset: pd.DataFrame
+    pipeline: "Pipeline"
```

Comment:
I needed to do forward declaration here, otherwise we're in for some dependency cycle fun!